bentoml/cli/__init__.py

```diff
@@ -346,6 +346,7 @@ def serve_gunicorn(
                     port=port,
                     num_of_workers=microbatch_workers,
                     prometheus_lock=prometheus_lock,
+                    model_server_workers=workers,
```

Comment:
is it possible to start GunicornBentoServer before  GunicornMarshalServer? that way you should be able to get the actual worker count via `gunicorn_app.cfg.workers`

Reply:
`gunicorn_app.run` will block itself.

Reply:
Ohh got it

Reply:
I think a slightly better way to do this is:
```
model_server_workder_count = workers or get_gunicorn_num_of_workers()
```
and then pass it to both GunicornBentoServer and GunicornMarshalServer

That also fixes a current issue, for example when user set the CLI arg `--workers=5`, it still prints something like "get_gunicorn_num_of_workers: 3, calculated by cpu count", which is wrong and very confusing

Reply:
Oh yeah, great!